gitlab-pipeline/stage/yocto-build-n-test.yml

```diff
@@ -57,8 +57,9 @@ test:acceptance:qemux86_64:uefi_grub:
     CROSS_PLATFORM_TESTS_ARG: "--no-cross-platform-tests"
   script:
     - cd ${CI_PROJECT_DIR}
-    - mkdir -p stage-artifacts
+    - mkdir -p stage-artifacts/qemu
     - cp $WORKSPACE/qemux86-64-uefi-grub/qemux86-64-uefi-grub_release_1_*.mender stage-artifacts/
+    - cp $WORKSPACE/qemux86-64-uefi-grub/{*.uefiimg,*.sdimg,*.wic,ovmf.qcow2,bzImage} stage-artifacts/qemu/
```

Comment:
I don't think we need a full copy here. Let's do `cp -l`, `cp -s` or `cp --reflink` (the first one that works).